__tests__/calendar-restful-model-collection-spec.js

```diff
@@ -166,6 +166,60 @@ describe('CalendarRestfulModelCollection', () => {
     });
   });
 
+  test('[CONSECUTIVE AVAILABILITY] should fetch results with params', done => {
+    const params = {
+      startTime: '1590454800',
+      endTime: '1590780800',
+      interval: 5,
+      duration: 30,
+      emails: [{ name: 'Jane', email: 'jane@email.com' }],
+      open_hours: [
+        {
+          emails: [{ name: 'Nylas', email: 'swag@nylas.com' }],
```

Comment:
similar here - plus you'll get an error of the open_hours contains an email that does not exist in the email or free_busy lists

Reply:
unless there is some other parsing step idk

Reply:
@tbadun thank you for letting me know about that error, I'll look into maybe having a check to ensure that open_hours emails all match the emails as found in the emails element in the main object. Is the email in the `open_hours` also a 2D array for consecutive?

Reply:
in open_hours emails will just be a list/1D array

Reply:
Fantastic, thanks!